### DISCUSSION TITLE
Version 9.10.2

### DISCUSSION THREAD

**Discussion by leelaisheng88** [dnnsoftware/Dnn.Platform#5167] [Q&A]
What does the version 9.10.2 component consists of? What will be the version for JQuery and JQuery_UI?

**Comment by mitchelsellers** [comment]
9.10.2 is already released and the release notes are available under the releases section.

**Comment by leelaisheng88** [comment]
It never mentioned on the version for JQuery and JQuery_UI

Email ID<https://efs-intranet.enterprisesg.gov.sg/?xmail-id=30e9c166-b252-4ae5-8ca3-2e6cbf86e03e>
This email has been filed into the sender's Personal folder.
[3S-id=30e9c166-b252-4ae5-8ca3-2e6cbf86e03e:e5758c33]
